index.js

```diff
@@ -1,118 +1,235 @@
+// hint while you can complete this with only conditionals based on strings it may help to equate choice to a number
+
+
+// sytax expanded bc it was frying my brain...
+// 99% sure there's a more elegant solution in here. Excited to find it later.
+let rockPaperScissors = function ( playerChoice) {
+    let computerNumber = Math.random();
+    if ( computerNumber <= (1/3) ) {
+        computerChoice = "Rock";
+    } else if ( (1/3) < computerNumber && computerNumber <= (2/3) ) {
+        computerChoice = "Paper";
+    } else {
+        computerChoice = "Scissors"
+    }
+
+    if ( playerChoice === "Rock" || playerChoice === "rock" ) {
+        if ( computerChoice === "Rock" ) {
+            result = "ties";
+        } else if ( computerChoice === "Paper" ) {
+            result = "loses";
+        } else {
+            result = "wins";
+        }
+    } else if ( playerChoice === "Paper" || playerChoice === "paper" ) {
+        if ( computerChoice === "Rock" ) {
+            result = "wins";
+        } else if ( computerChoice === "Paper" ) {
+            result = "ties";
+        } else {
+            result = "loses";
+        }
+    } else if ( playerChoice === "Scissors" || playerChoice === "scissors" ){
+        if ( computerChoice === "Rock" ) {
+            result = "loses";
+        } else if ( computerChoice === "Paper" ) {
+            result = "wins";
+        }else {
+            result = "ties";
+        }
+    } else {
+        result = "cheated"
+    }
+    console.log("Player " + result );
+}
+rockPaperScissors("spock");
 
-  
-  
 
 /************************************************************** Task 5 **************************************************************/
 //Metric Converter
 //a. KM to Miles - should take the number of kilometers and convert it to the equal number of miles
 
-
-
+let metricConverter = function (miles) {
+    console.log(miles + " miles converts to " + (miles*1.60934) + " kilometers" )
+}
+metricConverter(32);
 
 
 //b. Feet to CM - should take the number of feet and convert it to the equal number of centimeters
-  
-
 
+let feetToCentimeters = function (feet){
+    console.log(feet + " feet converts to " + (feet*30.48) + " cm.")
+}
+feetToCentimeters(7);
 
 
 /************************************************************** Task 6 **************************************************************/
 // 99 bottles of soda on the wall
 // create a function called annoyingSong
 // the function should take a starting number as an argument and count down - at each iteration it should log (number) bottles of soda on the wall, (number) bottles of soda, take one down pass it around (number left over) bottles of soda on the wall`
-  
-
 
 
+let annoyingSong = function(startingNumber){
+    for(i = startingNumber; i > 0; i--){
+        console.log(i + " bottles of pop on the wall, " + i + " bottles of pop, take one down and pass it around, " + (i-1) + " bottles of pop on the wall")
```

Comment:
"pop"

Reply:
it just sounded so weird with "soda" in my head. Totally ruined the rhythm. I'll accept a point off for the sake of the music.